tsconfig.json

```
@@ -1,8 +1,23 @@
 {
-  "files": [],
-  "references": [
-    { "path": "./tsconfig.app.json" },
-    { "path": "./tsconfig.node.json" }
-  ],
-  "compilerOptions": { "baseUrl": ".", "paths": { "@src/*": ["src/*"] } }
+  "compilerOptions": {
+    "target": "ESNext",
+    "module": "ESNext",
+
+    "moduleResolution": "bundler",
+    "strict": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "jsx": "react-jsx",
+
+    "baseUrl": ".",
+    "paths": {
+      "@src/*": ["src/*"]
```

```suggestion
      "@/*": ["src/*"]
```

저두 다시 찾아봤는데 paths를 이렇게 작성해주면 사용할 때`@src/` 이게 아니라 `@/` 이렇게 적어줄 수 있는 것 같아요~!

---

`@/`로 수정해서 merge하겠습니다!